karols-frontend/src/components/ListChoiceGender.js

```diff
@@ -0,0 +1,11 @@
+import React from "react"
+import ChoiceGender from "./ChoiceGender"
+
+const ListChoiceGender = () => (
+  <div>
```

Comment:
You are displaying ListChoiceGender in ListChoiceGender why ? when ? correc display ?